var/spack/repos/builtin/packages/slate/package.py

```diff
@@ -44,6 +44,12 @@ def install(self, spec, prefix):
 
         compiler = 'mpicxx' if spec.variants['mpi'].value else ''
 
+        configfile = '{}/testsweeper/config/config.py'.format(self.stage.source_path)
+        filter_file('log\.write\( stdout \)','log.write( str(stdout) )', configfile, backup=False)
```

Comment:
```suggestion
        filter_file('log.write( stdout )', 'log.write( str(stdout) )', configfile, backup=False, string=True)
```
Same below, plus line wrapping to make flake8 happy